components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -120,6 +120,10 @@ scoped_refptr<Extension> ConvertGreaselionRuleToExtensionOnTaskRunner(
         ? extensions::manifest_values::kRunAtDocumentStart
         : extensions::manifest_values::kRunAtDocumentEnd);
 
+  if (!rule->messages().empty()) {
+    root->SetStringPath(extensions::manifest_keys::kDefaultLocale, "en_US");
```

Comment:
shouldn't we be using the browser default locale here?

Reply:
@bridiver I don't think so. Browser will use current locale to choose which translations to make. This field indicates the locale to use when there are no translations available for the user's language. https://developer.chrome.com/extensions/i18n